src/js/modules/sections/components/SectionPage.js

```diff
@@ -3,26 +3,121 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import injectSheet from 'react-jss';
-import sections from '../../sections';
+import { Grid, Row, Col } from 'react-bootstrap/lib/';
 
 import articles from '../../articles';
 
 const styles = {
   SectionPage: {
     color: '#000',
+  },
+  sectionName: {
+    width: '1080px',
+    height: '43px',
```

Comment:
You don't have to copy ALL the CSS from Zeplin. For example, why would you include the width and height of a textbox? The text will define itself. Moreover, some of the CSS here is just redundant. See lines 23 and 24: you set padding to '0px 0px', and then afterwards you set paddingLeft to '0px'.

`padding: '0px 0px'` is also redundant in itself. Know how to use <a href="https://www.w3schools.com/css/css_padding.asp">padding</a>, and also familiarize yourself with the <a href="https://www.w3schools.com/css/css_boxmodel.asp">box model</a>